kin-core/src/test/java/kin/core/AccountInfoRetrieverTest.java

```diff
@@ -86,7 +103,15 @@ public void getBalance_SameIssuerDifferentAsset_AccountNotActivatedException() t
     }
 
     @Test
-    public void getBalance_IOException() throws Exception {
+    public void getStatus_SameIssuerDifferentAsset_StatusNotActivated() throws Exception {
```

Comment:
what is the different between this test and getStatus_NoKinTrust_StatusNotActivated

Reply:
all these tests are similar to get balance tests, this one purpose is to verify i'm checking both issuer and asset code when deciding whether this account is activated (has trust line), in NoKinTrust I just don't have any trust in the account i'm tesing

Reply:
@yosriz the tests are identical and they also use the same "balance_res_no_kin_trust.json" file

Reply:
oh... now I see. I need to use a different json here 'balance_res_same_issuer_different_asset_code.json', the same goes for `getBalance_SameIssuerDifferentAsset_AccountNotActivatedException`